README.md

```diff
@@ -6,61 +6,255 @@ Cookiecutter template for new CDP deployments.
+    _CDP-Seattle monthly average: ~$3.00_
+-   [Google Speech-to-Text Pricing](https://cloud.google.com/speech-to-text/pricing)
+    _CDP-Seattle monthly average: ~$22.00_
+
+**Total Average Monthly Cost**: $33.00
+
+### Speech-to-Text
+
+You may not need to use speech-to-text! In the case your municipalicity provides closed
+caption files in a format we support parsing and cleaning, we can use those files
+instead of using speech-to-text. When using closed caption files for transcription
+generation. CDP-Seattle speech-to-text costs dropped to ~$2.00 / month.
+
+To use closed captions files instead of generating a transcript from audio,
+your event gather function can attach a
+[`closed_caption_uri`](https://councildataproject.github.io/cdp-backend/cdp_backend.pipeline.html#cdp_backend.pipeline.ingestion_models.Session)
+to the `Session` object.
+
+With speech-to-text cost removed, total average monthly cost for CDP-Seattle is ~$13.00.
+
+### Future Processing Features
+
+As we add more features to CDP that require additional processing or resources we will
+continue to try to minimize their costs wherever possible. Further, if a feature is
+optional, we will create a flag that maintainers can set to include or exclude the
+additional processing or resource usage.
+See [Upgrades and New Features](#upgrades-and-new-features) for more information.
+
+## Upgrades and New Features
+
+In general, all updates and upgrades are handled easily for you through
+automated systems that run using GitHub Actions on your repository.
+
+Upgrades are delivered either just before processing time (during package installation)
+or as a part of weekly system checks and auto-deployments.
+
+#### Backend Pipeline Upgrades
+
+Due to how we have constructed how event processing works, every time CDP
+pipelines are initiated, they also install the latest non-breaking-changes version of
+the [cdp-backend](https://github.com/CouncilDataProject/cdp-backend) package.
+
+In this way, every time your pipeline runs to gather events, index events, or
+index pieces of legislation, you can be sure you are up-to-date.
+
+#### Backend Infrastructure Upgrades
+
+Every week, your repository will automatically check the status of your infrastructure
+and deploy updates as detected.
+
+#### Frontend Web Application Upgrades
+
+Every week, your repository will automatically build and deploy the CDP web application,
+during the build process any non-breaking changes and new features will automatically be
+pulled in.
+
+### Breaking Changes
+
+In the case we release a new version of
+[cdp-backend](https://github.com/CouncilDataProject/cdp-backend) or
+[cdp-frontend](https://github.com/CouncilDataProject/cdp-frontend) that includes
+breaking changes we will also release an upgrade guide that you will be able to find
+in this repository.
+
+In the case we can fully automate the upgrade, we will include a script to do so that
+CDP Instance maintainers will simply have to run.
```

Comment:
Minor edit for readability, but I'm not sure where a maintainer might run an upgrade script, so feel free to edit that part. 

```suggestion
In the case we can fully automate the upgrade, we will include a script that
CDP Instance maintainers will simply have to run on their project.
```